static/js/alert_words_ui.js

```diff
@@ -20,14 +19,14 @@ function add_alert_word(word, event) {
     if (word === '') {
         return;
     }
-
     var final_li = templates.render('alert_word_settings_item', {'word': word, editing: false});
 
-    var li = $(event.target).parent();
+    // the li element is the 5th parent of the input form and button
+    var li = $(event.target).parents().eq(4);
```

Comment:
you should use e.g. `$(event.target).parents('li')` -- it's much more robust in the case that we refactor the HTML in the future.  (There's another instance of this below, too).